README.md

```diff
@@ -20,7 +20,7 @@ Self-Driving Car Engineer Nanodegree Program
 
 1. Clone this repo.
 2. Make a build directory: `mkdir build && cd build`
-3. Compile: `cmake .. && make`
+3. Compile: `cmake .. && make` (Windows: `cmake .. -G "Unix Makefiles" && make`)
```

Comment:
Nit: could we just make this a subbullet of step 3? Eg.

3. Compile: ...
  * On Windows, you may need to run: ...